client/src/containers/overview/project-details/parameters/index.tsx

```diff
@@ -69,8 +83,15 @@ export const PROJECT_PARAMETERS = [
   },
 ] as const;
 
+function useFilters() {
+  return useQueryState(
+    "filters",
+    parseAsJson(filtersSchema.parse).withDefault(INITIAL_FILTERS_STATE),
+  );
+}
+
```

Comment:
Let's keep the filters for detail really simple and not share them in the URL: this might be confusing for the user as we already sharing the filters of the overview page. Use a jotai's atom to handle them.